docs/installation.rst

```diff
@@ -0,0 +1,55 @@
+.. _installation:
+
+************
+Installation
+************
+
+Installing ``hvpy``
+-------------------
+
+If you are new to Python and/or do not have familiarity with `Python virtual environments <https://docs.python.org/3/tutorial/venv.html>`__, then we recommend starting by installing `miniforge <https://github.com/conda-forge/miniforge#miniforge3>`__.
+This works on all platforms (Linux, Mac, Windows) and installs a full-featured Python environment in your user directory without requiring root privileges.
+
+Using pip
+---------
+
+.. warning::
+
+    Users of any conda distribution should follow the instructions for :ref:`conda_install`.
```

Comment:
I don't think this warning is needed since there's only one step of pip install. Someone on the page would see the pip and condo sections at the same time so they shouldn't need a warning telling them to jump to their section.